resources/units/aircraft/EA-18G.yaml

```diff
@@ -0,0 +1,41 @@
+carrier_capable: true
+description:
+  "The Boeing EA-18G Growler is an American carrier-based electronic warfare aircraft,
+  a specialized version of the two-seat F/A-18F Super Hornet. The EA-18G replaced the
+  Northrop Grumman EA-6B Prowlers in service with the United States Navy."
+introduced: 1999
+manufacturer: McDonnell Douglas
+origin: USA
+price: 25
+role: Carrier-based Multirole Fighter
+fuel:
```

Comment:
Oh, and as on the last review: I expect the super hornet and legacy hornet do not have the same fuel flow. No data is better than wrong data.

Reply:
In DCS they actually do because it's the same flight model.

Reply:
Huh, okay then.